jupyros/ros1/ros_widgets.py

```diff
@@ -316,7 +316,7 @@ def step_clk(arg):
     return vbox
 
 
-def client(srv_name, srv_type):
+def service_client(srv_name, srv_type):
```

Comment:
if we wanted to, we could add a deprecation warning to the old function (just in case people have old code lying around) :)

Reply:
e.g.

def client(*args, **kwargs):
    from warnings import warn
    warn("get with the program!")
    return service_client(*args, **kwargs)

Reply:
That's a good idea! I'll give it a try